other-docs/guides/how-to-submit-a-bug-report.md

```diff
@@ -0,0 +1,19 @@
+# How to submit a bug report.
+
+Bugs happen, and we’re here to support you, should you find a bug in Altis.
+
+To get the most out of support and to help our product team, there are some requirements for a bug report that must be met, in order for us to help you with your issue.
+
+When filing a bug report ( [you can read this guide on how to file support tickets](https://docs.altis-dxp.com/nightly/guides/getting-help-with-altis/) ) please provide all of the following:
+## Test Results.
+
+Please write a brief description of the bug, including what you expect to happen and what is currently happening.
+## [Reduced Test Case](https://css-tricks.com/reduced-test-cases/).
+
+Please ensure you test the bug in as a reduced test case, and provide the reduced test case as the basis of the bug report.
```

Comment:
```suggestion

## Reduced Test Case

Bugs should be isolated to a [reduced test case](https://css-tricks.com/reduced-test-cases/), as this helps us to ensure that the bug exists in Altis, rather than your project. Reduced test cases should be created against a standard Altis local environment with just the minimal changes needed to reproduce the bug.

Please ensure you provide the reduced test case as the basis of the bug report. If we cannot confirm that the test case is within Altis, we may ask you to reduce the test case further to ensure it isn't caused by your custom code.

Customers on Enterprise support plans may wish to use their dedicated technical engineering support to handle bug isolation.

```